indexclient/client.py

```diff
@@ -305,21 +311,25 @@ def list_versions(self, did):
             versions.append(Document(self, version["did"], version))
         return versions
 
+    @indexd_request_wrapper
     def _get(self, *path, **kwargs):
         resp = requests.get(self.url_for(*path), **kwargs)
         handle_error(resp)
         return resp
 
+    @indexd_request_wrapper
     def _post(self, *path, **kwargs):
```

Comment:
I'm still having concerns adding retries on `POST`, `PUT` or `DELETE` - it is totally fine to add timeouts, but retrying means any future IndexD APIs must be strictly repeatable (calling with the same args for more than one times ends up with identical results), if current APIs are already repeatable. What was the initial motivation to retry on e.g. `POST` please?

Reply:
I have a ticket to copy and index million files. We have to make sure that all the new files need to be indexed in indexd. For safety, I think I will only add `timeout` for `POS`, `PUT` and `DELETE` and change the logic in my code. Thank you for your comment!
Fixed in 5f39c8bcb7a47e5052033c6ad2b7d0e3df0f5b72